src/devtools/client/debugger/src/components/Editor/LineHitCounts.module.css

```diff
@@ -3,19 +3,26 @@
   align-items: center;
   justify-content: center;
 
-  width: 26px;
-  height: var(--theme-code-font-size);
-  margin: 2px;
-  padding-top: 1px;
+  width: 100%;
+  height: calc(var(--theme-code-font-size) * 1.4);
```

Comment:
Ooo, why 1.4? That seems...kind of random 😁

Reply:
Yeah, I should have put a comment here. It's sort of random, it's to remove the borders between each line background. Hopefully, we can look into a better way to handle this through CodeMirror.